src/module-1/card/atocart.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
consider having folders for images and icons that can be reused between card and cards list